src/ClientCommand.cpp

```diff
@@ -635,20 +635,19 @@ void CClient::UserCommand(CString& sLine) {
 			return;
 		}
 
-		const CModules& vMods = pOldNetwork->GetModules();
-		for (CModules::const_iterator i = vMods.begin(); i != vMods.end(); ++i) {
-			CString sOldModPath = pOldNetwork->GetNetworkPath() + "/moddata/" + (*i)->GetModName();
-			CString sNewModPath = pNewUser->GetUserPath() + "/networks/" + sNewNetwork + "/moddata/" + (*i)->GetModName();
+		for (auto pMod : pOldNetwork->GetModules()) {
```

Comment:
@jpnurmi my reasoning was that in a loop it should be really clear what the value is.